as/cNFT/assembly/models/persistent_nft_contract_metadata.ts

```diff
@@ -17,6 +17,7 @@ export class NFTContractMetadata {
     render_script: string
     style_css: string
     parameters: string
+    nft_per_owner: number
```

Comment:
Also would rename this `mints_per_address` cause it's not strictly how many nfts one address can own but how many can be minted, which is fine for our use case but should be renamed.